templates/macros/assets.html

```diff
@@ -1,56 +1,94 @@
-        {% endif %}
-        <div class="card-text">
-            <div class="card-title">{{ post.title }}</div>
-            <div class="card-description">
-                {{ post.description | striptags | truncate(length=100) | safe }}
+        <div class="asset-card__description">
+            {{ post.description | striptags | truncate(length=120) | safe }}
+        </div>
+        <div class="asset-card__tags">
+            {% if post.extra.bevy_versions %}
+            <div class="asset-card__tag-list asset-card__bevy-versions">
+                <img src="/assets/bevy_icon_dark.svg" alt="Small bevy icon">
+                {% for version in post.extra.bevy_versions %}
+                <span class="asset-card__tag">{{version}}</span>
+                {% endfor %}
+            </div>
+            {% endif %}
+            {% if post.extra.licenses %}
+            <div class="asset-card__tag-list asset-card__licenses">
+                {{ assets_macros::license_logo_svg() }}
+                {% for license in post.extra.licenses %}
+                <span class="asset-card__tag">{{license}}</span>
+                {% endfor %}
             </div>
+            {% endif %}
         </div>
     </a>
 </div>
 {% endmacro card %}
 
 {% macro assets_menu_row(prefix, section_path) %}
-    {% set section = get_section(path=section_path) %}
-    {% set id = prefix ~ '-' ~ section.path | slugify %}
-    {% set label_class = "tree-menu__label" %}
+{% set section = get_section(path=section_path) %}
+{% set id = prefix ~ '-' ~ section.path | slugify %}
+{% set label_class = "tree-menu__label" %}
 
-    {% if section.subsections %}
-        {% set label_class = label_class ~ " tree-menu__label--with-chevron" %}
-        <input id="{{id}}" class="tree-menu__state" type="checkbox" checked>
-    {% endif %}
+{% if section.subsections %}
+{% set label_class = label_class ~ " tree-menu__label--with-chevron" %}
+<input id="{{id}}" class="tree-menu__state" type="checkbox" checked>
+{% endif %}
 
-    <li class="tree-menu__item">
-        <div class="{{label_class}}">
-            <a class="tree-menu__link" href="#{{section.title | slugify}}">{{ section.title }}</a>
-            {% if section.subsections %}
-                <label class="tree-menu__toggle" for="{{id}}">
-                    <img class="tree-menu__chevron" src="/assets/icon-chevron-down.svg">
-                </label>
-            {% endif %}
-        </div>
+<li class="tree-menu__item">
+    <div class="{{label_class}}">
+        <a class="tree-menu__link" href="#{{section.title | slugify}}">{{ section.title }}</a>
         {% if section.subsections %}
-            <ul class="tree-menu">
-                {% for s in section.subsections %}
-                    {{ self::assets_menu_row(prefix=prefix, section_path=s) }}
-                {% endfor %}
-            </ul>
+        <label class="tree-menu__toggle" for="{{id}}">
+            <img class="tree-menu__chevron" src="/assets/icon-chevron-down.svg">
```

Comment:
This is missing an `alt` attribute and should maybe have a `title` attribute to indicate that you can toggle something here (if I understand this correctly)